examples/olm/catalog-create-subscribe.yaml

```diff
@@ -33,5 +33,5 @@ spec:
   channel: stable
   installPlanApproval: Automatic
   name: local-storage-operator
-  source: hekumar-operators
+  source: redhat-local-storage-src
```

Comment:
Do we also want to change line https://github.com/openshift/local-storage-operator/blob/2bd3bbed154cb80e90de35f83460f80a66db9572/examples/olm/catalog-create-subscribe.yaml#L23?

Reply:
it has to be this still, because the operator is published as a application to my quay.io account. We just moved from using catalog registry which was hosted under my account to using an application still hosted under my account in quay.io